README.md

```diff
@@ -162,11 +193,181 @@ $ helm delete my-splunk-otel-collector
 The command removes all the Kubernetes components associated with the chart and
 deletes the release.
 
-## Advanced Configuration
+### Advanced Configuration
 
 To fully configure the Helm chart, see the [advanced
 configuration](docs/advanced-configuration.md).
 
+## Setup Splunk
+
+Splunk OpenTelemetry Collector for Kubernetes supports installation using Helm. Read the Prerequisites, Installation and Deployment documentation before you start your deployment of Splunk OpenTelemetry Collector for Kubernetes.
+
+Perform the following steps before you install:
+
+1. Create a minimum of one Splunk platform index. This index events index, which will handle logs.
+If you do not configure this index, Splunk OpenTelemetry Collector for Kubernetes uses the defaults created in your HTTP Event Collector (HEC) token.
+
+2. Create a HEC token if you do not already have one. If you are installing the connector on Splunk Cloud, file a ticket with Splunk Customer Service and they will deploy the indexes for your environment, and generate your HEC token.
+
+### Setup for Non-Root User Group
```

Comment:
remove this part `Setup for Non-Root User Group`

Reply:
it is covered in advanced configuration.

Reply:
removed